packages/template-typescript-minimal/.force_overrides

```diff
@@ -0,0 +1,11 @@
+// DISCLAIMER 
+//
+// BY USING THIS FILE, YOU AGREE THAT THE FUNCTIONALITY OF YOUR INSTALLED EXTENSION(S) IS NOT GUARANTEED.
+// ADDITIONALLY UPGRADABILITY OF EXTENSIONS CAN ALSO NO LONGER BE GUARANTEED AND IS NOT SUPPORTED BY SALESFORCE. 
+// USE ONLY AS A TEMPORARY SOLUTION TO URGENTLY PATCH/UPDATE AN EXTENSION.
+//
+// USAGE: 
+// PLACE THE RELATIVE __POSIX__ PATH TO THE EXTENSION FILE YOU WANT TO OVERRIDE STARTING WITH THE EXTENSION PACKAGE NAME.
+// MULTIPLE OVERRIDES CAN BE ADDED TO THIS FILE, ONE PER LINE.\
+// EXAMPLE:
+// @salesforce/extension-sample/src/pages/home.tsx
```

Comment:
🤔 are the file paths supposed to be relative paths or absolute paths? The example doesn't look like either

Reply:
So it's really more of a module import path, I was wavering on this a little too. I could make it something like:

`./@salesforce/extension-sample/src/pages/home.tsx` 

or 

`./node_modules/@salesforce/extension-sample/src/pages/home.tsx`

I think the second one is probably the way to go and shouldn't be hard to fix.